package.json

```diff
@@ -37,7 +37,7 @@
     "command-node": "0.1.1",
     "request": "2.88.0",
     "async": "1.5.2",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#feature/842_ngsi_ld",
```

Comment:
We need a strategy for this. Maybe keep the PR open until the NGSI-LD branch is merged in the node lib and then merge this PR?

Reply:
Yes I share the same view as you do. Let's wait for iotagent-node-lib to be merged before trying to merge anything.